tests/Integrations/PHPRedis/V5/PHPRedisClusterTest.php

```diff
@@ -1750,6 +1751,57 @@ public function getBinarySafeString()
         return $binarySafeString;
     }
 
+    public function testSplitByDomainWithClusterName()
+    {
+        $this->putEnvAndReloadConfig(['DD_TRACE_REDIS_CLIENT_SPLIT_BY_HOST=true']);
+        $redis = new \RedisCluster('redis_integration', [
+            \implode(':', $this->connection1),
+            \implode(':', $this->connection2),
+            \implode(':', $this->connection3),
+        ]);
```

Comment:
Can you please include the rediscluster instantiation in the isolateTracer() block to test that particular span as well?